template.tpl

```diff
@@ -66,14 +59,6 @@ const copyFromWindow = require('copyFromWindow');
 const callInWindow = require('callInWindow');
 const createQueue = require('createQueue');
```

Comment:
do we want to keep these definitions if they're no longer being used?

Reply:
duh, good call! just removed